platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -343,7 +343,7 @@ In style specification | Method, function, or predicate type | Format string syn
 `properties`           | |`$mgl_featureProperties`
 `at`                   | `objectFrom:withIndex:` | `array[n]`
 `get`                  | `+[NSExpression expressionForKeyPath:]` | Key path
-`has`                  | `mgl_hasProperty:properties:` |
+`has`                  | `mgl_does:have:` | `mgl_does:have:(self, 'key')`
```

Comment:
I renamed and reordered this function. Instead of calling `mgl_hasProperty:properties:('key', self)`, you call `mgl_does:have:(self, 'key')`. I think this reads better: “Does self have ‘key’?” It’s also consistent with the order when you call the conventional function: `FUNCTION(self, 'mgl_has:', 'key')`.